MSAL/test/app/mac/MSALAcquireTokenViewController.m

```diff
@@ -72,6 +72,10 @@ @implementation MSALAcquireTokenViewController
 - (void)viewDidLoad
 {
     [super viewDidLoad];
+    //TODO: MSAL MacOS test app doesn't support embedded webview mode
```

Comment:
I think you'll need Peter's PR to complete before you can merge this.

Reply:
@Peter will merge his PR on top on this. Already discussed with him offline.